drivers/net/slip.c

```diff
@@ -434,6 +440,7 @@ static void slip_iface_init(struct net_if *iface)
 
 static struct slip_context slip_context_data;
 
+#if defined(CONFIG_SLIP_TAP) && defined(CONFIG_NET_L2_ETHERNET)
```

Comment:
The `&& defined(CONFIG_NET_L2_ETHERNET)` should not be needed as the CONFIG_SLIP_TAP selects it

Reply:
I didn't add this line, I simply moved it but sure, will change that.

Reply:
Better to check for TAP, as you could have Ethernet enabled but SLIP configured to use TUN.

Reply:
Ok, done.